components/colors/colors.js

```diff
@@ -17,86 +17,80 @@ if (!document.head.querySelector('#d2l-colors')) {
+			--d2l-color-fluorite-plus-2: #f9ebff;
+			--d2l-color-fluorite-plus-1: #ce68fa;
+			--d2l-color-fluorite: #9d1fd4;
+			--d2l-color-fluorite-minus-1: #6900a0;
 
 			/* tourmaline */
-			--d2l-color-tourmaline-plus-2: #fae1ed;
-			--d2l-color-tourmaline-plus-1: #ff389b;
-			--d2l-color-tourmaline: #bf2a75;
-			--d2l-color-tourmaline-minus-1: #96215c;
+			--d2l-color-tourmaline-plus-2: #ffebf6;
+			--d2l-color-tourmaline-plus-1: #fd4e9d;
+			--d2l-color-tourmaline: #d40067;
+			--d2l-color-tourmaline-minus-1: #990056;
 
 			/* cinnabar */
-			--d2l-color-cinnabar-plus-2: #fae1e2;
-			--d2l-color-cinnabar-plus-1: #ff6b70;
+			--d2l-color-cinnabar-plus-2: #ffede8;
+			--d2l-color-cinnabar-plus-1: #ff575a;
 			--d2l-color-cinnabar: #cd2026;
-			--d2l-color-cinnabar-minus-1: #a1191d;
+			--d2l-color-cinnabar-minus-1: #990006;
 
 			/* carnelian */
-			--d2l-color-carnelian-plus-1: #fae3cf;
+			--d2l-color-carnelian-plus-1: #fff3e0;
 			--d2l-color-carnelian: #e87511;
-			--d2l-color-carnelian-minus-1: #a6540d;
-			--d2l-color-carnelian-minus-2: #804008;
+			--d2l-color-carnelian-minus-1: #ba4700;
+			--d2l-color-carnelian-minus-2: #7d2600;
 
 			/* citrine */
-			--d2l-color-citrine-plus-1: #fae5c8;
+			--d2l-color-citrine-plus-1: #fff9d6;
 			--d2l-color-citrine: #ffba59;
-			--d2l-color-citrine-minus-1: #a87b3b;
-			--d2l-color-citrine-minus-2: #694b25;
-
-			/* topaz */
-			--d2l-color-topaz-plus-1: #fffbb2;
-			--d2l-color-topaz: #f5ec5a;
-			--d2l-color-topaz-minus-1: #a69f21;
-			--d2l-color-topaz-minus-2: #635f15;
+			--d2l-color-citrine-minus-1: #c47400;
+			--d2l-color-citrine-minus-2: #7a4300;
 
 			/* peridot */
-			--d2l-color-peridot-plus-1: #cdf2b1;
-			--d2l-color-peridot: #8ccd5a;
-			--d2l-color-peridot-minus-1: #59823a;
-			--d2l-color-peridot-minus-2: #3d5926;
+			--d2l-color-peridot-plus-1: #efffd9;
+			--d2l-color-peridot: #8ad934;
+			--d2l-color-peridot-minus-1: #4a8f00;
+			--d2l-color-peridot-minus-2: #2f5e00;
 
 			/* olivine */
-			--d2l-color-olivine-plus-1: #a5fabd;
+			--d2l-color-olivine-plus-1: #e7ffe3;
 			--d2l-color-olivine: #46a661;
-			--d2l-color-olivine-minus-1: #327846;
-			--d2l-color-olivine-minus-2: #275c36;
+			--d2l-color-olivine-minus-1: #027a21;
+			--d2l-color-olivine-minus-2: #005614;
 
 			/* malachite */
-			--d2l-color-malachite-plus-1: #a1f7f4;
-			--d2l-color-malachite: #00afaa;
-			--d2l-color-malachite-minus-1: #017d79;
-			--d2l-color-malachite-minus-2: #005e5b;
+			--d2l-color-malachite-plus-1: #e3fff5;
+			--d2l-color-malachite: #2de2c0;
+			--d2l-color-malachite-minus-1: #00a490;
+			--d2l-color-malachite-minus-2: #00635e;
 
 			/* primary accent */
 			--d2l-color-primary-accent-action: var(--d2l-color-celestine);
 			--d2l-color-primary-accent-indicator: var(--d2l-color-carnelian);
 
 			/* feedback */
 			--d2l-color-feedback-error: var(--d2l-color-cinnabar);
-			--d2l-color-feedback-warning: var(--d2l-color-citrine);
+			--d2l-color-feedback-warning: var(--d2l-color-carnelian);
```

Comment:
In the feedback screenshot, it looks like the warning bit should probably use white text now that the background is darker.